photo-gallery/src/App.js

```diff
@@ -21,21 +21,24 @@ export default class App extends Component {
     return (
       <div className="App">
         <Header />
-        <select onChange={handleChange}>
-          <option value="" defaultValue>All</option>
-          <option value="narwhal">Narwhal</option>
-          <option value="rhino">Rhino</option>
-          <option value="unicorn">Unicorn</option>
-          <option value="unilego">Unilego</option>
-          <option value="triceratops">Triceratops</option>
-          <option value="markhor">Markhor</option>
-          <option value="mouflon">Mouflon</option>
-          <option value="addax">Addax</option>
-          <option value="chameleon">Chameleon</option>
-          <option value="lizard">Lizard</option>
-          <option value="dragon">Dragon</option>
-        </select>
-        {<ImageList image={filteredImages} />}
+        <main>
+          <label for="keyword">Filter by Keyword:</label>
```

Comment:
So looks liek you had an error in your console that said something like improper use of for in a label tag.
to fix this all you need to do is htmlFor instead of for, its a react specific thing!